stdlib/time/src/main/ballerina/src/time/timelib.bal

```diff
@@ -282,8 +364,11 @@ function externToTimeZone(Time time, handle zoneId) returns Time|Error = @java:M
 } external;
 
 # Returns the current time value with the system default time-zone.
+# ```ballerina
+#  time:Time now = time:currentTime();
+# ```
 #
-# + return - Time object containing the time and zone information
+# + return - Time object containing the time and the zone information
```

Comment:
```suggestion
# + return - Time object containing the time and zone information
```

Reply:
Zone is a different type of information from time.